scripts/run-executable.ps1

```diff
@@ -227,11 +264,39 @@ function Start-Executable {
             $pinfo.Arguments = "--modules=$(Split-Path $Path -Parent) --cover_children --sources src\core --excluded_sources unittest --working_dir $($LogDir) --export_type binary:$(Join-Path $CoverageDir $CoverageName) -- $($Path) $($Arguments)"
             $pinfo.WorkingDirectory = $LogDir
         } else {
-            $pinfo.FileName = $Path
-            $pinfo.Arguments = $Arguments
-            # Enable WER dump collection.
-            New-ItemProperty -Path $WerDumpRegPath -Name DumpType -PropertyType DWord -Value 2 -Force | Out-Null
-            New-ItemProperty -Path $WerDumpRegPath -Name DumpFolder -PropertyType ExpandString -Value $LogDir -Force | Out-Null
+            Write-Host "Configuring process to collect crash dumps to $LogDir"
+
+            if ($UseProcDump) {
+                # Use ProcDump to launch and monitor the process
+                try {
+                    $pd = Ensure-ProcDump
+                    Write-Host "Using ProcDump to launch and monitor process"
+
+                    # ProcDump will launch the target process and monitor it
+                    # -accepteula: auto-accept EULA
+                    # -ma: full memory dump
+                    # -e: dump on unhandled exception
+                    # -x <dir>: write dumps to directory
+                    # Then: executable path and arguments
+                    $pinfo.FileName = $pd
+                    $pinfo.Arguments = "-accepteula -ma -e -x `"$LogDir`" `"$Path`" $Arguments"
+                    Write-Host "ProcDump command: $($pinfo.FileName) $($pinfo.Arguments)"
+                } catch {
+                    Write-Warning "Failed to setup ProcDump: $_"
+                    Write-Warning "Falling back to direct execution without crash dumps"
+                    $pinfo.FileName = $Path
+                    $pinfo.Arguments = $Arguments
+                }
+            } else {
+                # Direct execution with WER
```

Comment:
If WER is broken and not expected to be fixed, should we simply simplify the script by removing support and rely on procdump only?

Reply:
sounds good